docs/examples/ai/resources/runDir/run_this.sh

```diff
@@ -1,4 +1,3 @@
-aws codeartifact login --tool pip --domain superai --repository pypi-superai-internal
```

Comment:
I guess you need to still do `aws codeartifact login --tool pip --domain superai --repository pypi-superai` externals have access to this

Reply:
That is done internally in the s2i script. So it's not required to explicitly specify the codeartifact login here.